src/components/Summary/Summary.scss

```diff
@@ -0,0 +1 @@
+@import 'global.scss';
```

Comment:
This is not used anymore, you must've still had it in your local branch. Let's delete this file. We're using `Summary.module.scss`